CDCgov/NEDSS-Modernization
# CNFT2-1460 Persist page size and pagination on pages library

## Description

Previously we added query params to track page number, this PR adds the same functionality for the page size, but also moves it to the range toggle so that all tables with pagination will have this functionality.

## Tickets

* [Jira Ticket](https://cdc-nbs.atlassian.net/jira/software/c/projects/CNFT2/boards/99?assignee=712020%3A591f5469-427d-49e9-87dc-8b4df72a02b4)

## Steps to verify

1. Navigate to /page-builder/manage/pages
2. Use the pagination alongside the page size selector.
3. Ensure when navigating between pages that the size persists.
4. Ensure when refreshing the page that both the size and page number persist.
5. Click into a page, and use the back button, ensure the previous page number and size remain.
6. When changing the size, the page number should revert to '1'
